public/index.html

```diff
@@ -9,6 +9,11 @@
     <meta name="theme-color" content="#000000" />
     <meta property="og:type" content="website" />
     <meta name="twitter:site" content="@ReactPlayIO" />
+    
+    <meta name="description" content="__META_DESCRIPTION__"/>
```

Comment:
Why do we need these as the branch changes are without meta?

Reply:
removed